src/GiphyClient.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Eve\Command;
+
+use GuzzleHttp\Client;
+
+class GiphyClient
+{
+    private $client;
+
+    public function __construct()

```

Comment:
As mentioned before, I'd prefer to override the constructor (and static factory method) to allow a GiphyClient to be injected, instead of instantiating it directly. Should anyone ever have a burning desire to write tests for this, it's going to be easier to mock/stub injected dependencies.